src/Compilers/CSharp/Portable/Syntax/LambdaUtilities.cs

```diff
@@ -57,7 +62,7 @@ public static SyntaxNode GetLambda(SyntaxNode lambdaBody)
         internal static SyntaxNode TryGetCorrespondingLambdaBody(SyntaxNode oldBody, SyntaxNode newLambda)
         {
             var oldLambda = oldBody.Parent;
```

Comment:
Agree. Thanks!

---
In reply to: [134011784](https://github.com/dotnet/roslyn/pull/21599#discussion_r134011784) [](ancestors = 134011784)